docker/docker-compose-scopes-example.yml

```diff
@@ -0,0 +1,87 @@
+  # Under the hood, this channel is implemented by encode/broadcaster (see link below).
+  # At the moment, the broadcast channel can be either: postgresdb, redis or kafka.
+  # The format of the broadcaster URI string (the one we pass to opal server as `OPAL_BROADCAST_URI`) is specified here:
+  # https://github.com/encode/broadcaster#available-backends
+  broadcast_channel:
+    image: postgres:alpine
+    environment:
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+  redis:
+    image: redis
+    ports:
+      - "6379:6379"
+  opal_server_worker:
+    image: permitio/opal-server:next
+    environment:
+      - OPAL_REDIS_URL=redis://redis:6379
+      - OPAL_SCOPES=1
+      - OPAL_SERVER_URL=http://opal_server:7002
+      - OPAL_WORKER_TOKEN=token
+      - OPAL_POLICY_REPO_POLLING_INTERVAL=30
+      - OPAL_BASE_DIR=/opal
+    volumes:
+      - opal:/opal:rw
+    depends_on:
+      - redis
+    command: sh -c "celery -A opal_server.worker worker -B -l debug -s /tmp/schedule.db"
+  opal_server:
+    # by default we run opal-server from latest official image
+    image: permitio/opal-server:next
+    environment:
+      # the broadcast backbone uri used by opal server workers (see comments above for: broadcast_channel)
+      - OPAL_BROADCAST_URI=postgres://postgres:postgres@broadcast_channel:5432/postgres
+      # number of uvicorn workers to run inside the opal-server container
+      - UVICORN_NUM_WORKERS=4
+      # the git repo hosting our policy
+      # - if this repo is not public, you can pass an ssh key via `OPAL_POLICY_REPO_SSH_KEY`)
+      # - the repo we pass in this example is *public* and acts as an example repo with dummy rego policy
+      # - for more info, see: https://github.com/permitio/opal/blob/master/docs/HOWTO/track_a_git_repo.md
+      - OPAL_POLICY_REPO_URL=https://github.com/permitio/opal-example-policy-repo
+      # in this example we will use a polling interval of 30 seconds to check for new policy updates (git commits affecting the rego policy).
+      # however, it is better to utilize a git *webhook* to trigger the server to check for changes only when the repo has new commits.
+      # for more info see: https://github.com/permitio/opal/blob/master/docs/HOWTO/track_a_git_repo.md
+      - OPAL_POLICY_REPO_POLLING_INTERVAL=30
+      # configures from where the opal client should initially fetch data (when it first goes up, after disconnection, etc).
+      # the data sources represents from where the opal clients should get a "complete picture" of the data they need.
+      # after the initial sources are fetched, the client will subscribe only to update notifications sent by the server.
+      - OPAL_DATA_CONFIG_SOURCES={"config":{"entries":[{"url":"http://host.docker.internal:7002/policy-data","topics":["policy_data"],"dst_path":"/static"}]}}
+      - OPAL_LOG_FORMAT_INCLUDE_PID=true
+      - OPAL_REDIS_URL=redis://redis:6379
```

Comment:
And again - I would strongly advice for merging this variable (`OPAL_REDIS_URL`) with `OPAL_BROADCAST_URI`.

We don't have to implement scopes for more DBs - we can simply validate that it's redis if scopes are enabled.
I don't like at all having to configure a backend DB twice just because different developers wrote the feature that uses it :)